serviceinsight/upgrades/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Upgrade Tips
+summary: Tips for upgrading to newer versions of ServiceInsight.
+reviewed: 2022-08-04
+related:
+---
+
+1. Ensure the machine running ServiceInsight can reach the service running ServiceControl and has access to the IP/Port.
```

Comment:
Is there no other page these could be included on?

Think of how many additional pages for the review process we're going to have.

Reply:
Moved this on to the support policy page